NEWS

```diff
@@ -1,5 +1,114 @@
 Geany 2.1 (unreleased)
 
+    General
+    * Improve activation of a running instance (Kondor Dániel, Issue#3531, PR#4071).
+    * Require GLib 2.56 (PR#4199).
+    * Include themes from geany-themes (Issue#4035, PR#4044).
+    * Change placeholders for external printing command (Pynckels, PR#4250).
+
+    Bug Fixes
+    * Fix Meson build when either fnmatch() or regcomp() is missing on the
+      system (Issue#3618).
+    * Do no silently truncate files containing NUL bytes (PR#3716).
+    * Improve many corner-case encoding detection and conversion (PR#3716).
+    * Fix search entry after editing the toolbar (Issue#3831, PR#3838).
+    * Fix possible invalid memory access when generating user initials on first
+      startup (Issue#3844, PR#3846).
+    * Fix non-deterministic anonymous tag names (Issue#3717, PR#3785).
+    * Fix loosing session when opening several projects (Issue#3897, PR#3898).
+    * Fix spurious CPU load after running a build command (PR#3902).
+    * Fix emission of the "document-activate" signal, leading to glitches in
+      UI and plugins (Issue#3684, PR#3891).
+    * Fix displaying symbols declared out of order (Issue#4060, PR#4063).
+
+    Interface
+    * Fix translation of the Various preferences tab (Issue#3628, PR#3634).
+    * Do not show redundant None encoding (Issue#3624, PR#3640).
+    * Move "Virtual Spaces" preference to "Editor -> Features" (Issue#3605).
+    * Increase default sidebar width (Issue#3630, PR#3637).
+    * Fix focusing the symbol list through key binding (Pascal Baillehache,
+      Issue#3678, PR#3681).
+    * Do not display encodings not supported on the system (PR#3720).
+    * Add support for searching the Messages and Compiler output (PR#3823).
+    * Add various pref to skip "replace in session" confirmation dialog
+      (Issue#3702, PR#3851).
+    * Add support for modern platform-native file selection dialogs (PR#3861).
+    * Fix spurious selection when clicking an externally-modified file
+      (Issue#3906, PR#3909).
+    * Make it easier to wrap search (Issue#1192, PR#4042).
+    * Regroup filetypes by letter (PR#3977).
+    * Make the Search & Replace dialog a little more compact (PR#4098).
+    * Make the Date dialog a little more compact (PR#4149).
+    * Add filter entry for keybindings preferences (Issue#2848, PR#4192).
+    * Add filter for open documents sidebar (PR#4210).
+
+    Editor
+    * Hide autocompletion and calltip popups when code is scrolled (PR#3560).
+    * Update Scintilla to 5.5.4 and Lexilla to 5.4.2 (PR#4168).
+    * Allow pasting and autocompletion for multiple cursors (Issue#625,
```

Comment:
cursors -> carets (I probably used the incorrect wording myself in the PR and commit title)